nodejs/week1/src/backend/routes/large-meals.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const meals = require("../data/meals.json");
+const router = express.Router();
+
+router.get("/", (req, res)=>{
+    largeMeals = meals.filter((largermeal)=>{
+        return largermeal.maxNumberOfGuests > 7;
+    });
```

Comment:
This can be simplified as

meals.filter(meal => meal.maxNumberOfGuests > 7)

Like with the cheap meals:

```
const largeMeals = meals.filter(meal => meal.maxNumberOfGuests > 7);
           /                      \
these are large          this takes every meal
because they                so not large yet, until
have been                         we make sure the max number of guests > 7
filtered
```